src/test/java/com/google/api/codegen/transformer/OutputTransformerTest.java

```diff
@@ -77,22 +80,15 @@ public void testAccessorNewVariableFailWithReservedKeyword() {
     when(namer.getAndSaveElementResourceTypeName(typeTable, resourceFieldConfig))
         .thenReturn("ShelfBookName");
     when(featureConfig.useResourceNameFormatOption(resourceFieldConfig)).thenReturn(true);
+    when(namer.getSampleUsedVarNames(context, form)).thenReturn(ImmutableSet.of("response"));
```

Comment:
suggestion: call this "fooresponse"  or something so it's obvious this is something that's only defined here and not in the code, and we an see this identifier clearly appear in the lines below.